### PR TITLE
Server.js

### PR DESCRIPTION
**Pull request by Josue650**
Finished the server.js

### CODE REVIEW

**Review by dolfo97 — APPROVED** [review]

File: server.js [hunk 1]
```diff
@@ -8,6 +8,36 @@ const MongoStore = require('connect-mongo')
...
+    session({
+        secret: process.env.SECRET,
+        store: MongoStore.create({ mongoUrl: process.env.MONGO_URI }),
+        saveUninitialized: true,
+        resave: false,
+    })
+)
+
+//* Start Middleware
+app.use(methodOverride('method'))
```
Comment (petermc87) [inline server.js]: Dont need this

File: server.js [hunk 2]
```diff
@@ -8,6 +8,36 @@ const MongoStore = require('connect-mongo')
...
 
+// Start Config //
+// This code makes us have req.body <=====
+app.use(express.urlencoded({ extended: true }))
+app.use((req, res, next) => {
+    res.locals.data = {}
+    next()
+})
+app.use(cors())
+app.engine('jsx', require('jsx-view-engine').createEngine())
```
Comment (petermc87) [inline server.js]: Dont need this

File: server.js [hunk 3]
```diff
@@ -8,6 +8,36 @@ const MongoStore = require('connect-mongo')
...
+app.set('view engine', 'jsx') // register the jsx view engine
```
Comment (petermc87) [inline server.js]: Dont need this

File: server.js [hunk 4]
```diff
@@ -8,6 +8,36 @@ const MongoStore = require('connect-mongo')
...
 const app = express()
 
+// Start Config //
+// This code makes us have req.body <=====
+app.use(express.urlencoded({ extended: true }))
+app.use((req, res, next) => {
+    res.locals.data = {}
+    next()
+})
+app.use(cors())
```
Comment (petermc87) [inline server.js:17]: Add `app.use(express.json())`

Comment (petermc87) [reply server.js:17]: Add above line 18

**Review by petermc87 — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (Josue650) [reply server.js]: fixed

File: server.js (hunk 4 above)
Comment (Josue650) [reply server.js:17]: Deleted non needed config and middleware

**Comment by Josue650** [comment]
Fixed server.js

**Review by petermc87 — APPROVED** [review]

**Review by dolfo97 — APPROVED** [review]

**Pull request merged**